CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/).
   - ?
 ### Changed
   - Updated Gradle to `6.6.1` version
+  - Each task will output reports into subdirectory inside `build/reports/ktlint` directory to fix non-working caching [#379](https://github.com/JLLeitschuh/ktlint-gradle/issues/379)
```

Comment:
The output file location is technically an API breaking change, if users are relying directly upon the output location, but from a compile standpoint it isn't.

What are your thoughts? Is this worthy of a major version bump?

Reply:
I think it ok to be minor release. Those who want old "behaviour" could redefine it to old approach.